src/ngraph_skip_assert.cc

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
```

Comment:
My suggestion is to use the term `disable_asserts` - so need to rename this file and function names accordingly.